### ISSUE TITLE
Handle internal computed vars

### ISSUE THREAD

**Issue by picklelo** [reflex-dev/reflex#1154]
We should be able to prefix computed vars with an underscore to make them internal computed vars.

```python
class State(pc.State):
    """The app state."""

    @pc.var
    def _now(self):
        return datetime.now()
```

**Comment by wassafshahzad** [comment]
Can I work on this ?

**Comment by picklelo** [comment]
@wassafshahzad yes, thanks for your contributions!

**Comment by wassafshahzad** [comment]
This issue can be closed

**Comment by masenf** [comment]
Fixed by #2540
